Vizzuality/widget-editor
# Standardise the select components

The main objective of this PR is to standardise the select components by re-using the same one everywhere.

<p align="center">
<img width="1416" alt="Screenshot of the editor with the new selects" src="https://user-images.githubusercontent.com/6073968/95897908-2b68cf80-0d86-11eb-8726-ead69dac8d76.png">
<p>

Additional changes include:
- Fixing an issue where the category/value selects would be displayed twice
- Showing the column options with their alias, description and type icon across the editor
- Displaying the aggregation function next to the selected column option (when it applies)
- Letting the user reset the “Order by” and “Value aggregation” selection (for the latter, the “None” option has been removed)
- Increasing the accessibility by connecting the selects to their label or adding an `aria-label` attribute
- Improving the position of the category/value selects relative to the chart

## Testing instructions

This PR is consequent. I would recommend testing the following cases:
- Checking the custom axis title work
- Applying an aggregation and making sure it appears in the value select (when collapsed)
- Checking if all the selects that list columns show the aliases, descriptions and type icons
- Checking the “Order by” and “Color” selects show the name of the aggregation (when the aggregated column is selected and the select is collapsed)
- Checking the category/value selects still work
- Checking the aggregation still works
- Checking the sorting (“Order by”) still works
- Checking the filters and end-user filters still work
- Checking the pie chart works and that the 3rd dimension for the other chart also does

For testing, you can use the following widget: `07b9cf1d-bdd1-4d1a-b849-14ad7f058af5` (dataset: `a86d906d-9862-4783-9e30-cdb68cd808b8`).

## Pivotal Tracker

- [Y axis difficult to read](https://www.pivotaltracker.com/story/show/172950921)
- [Standardisation of the inputs](https://www.pivotaltracker.com/story/show/175044722) (part of the task)
